deployment_scripts/config.ps1

```diff
@@ -3,7 +3,7 @@ $MONKEY_FOLDER_NAME = "infection_monkey"
 # Url of public git repository that contains monkey's source code
 $MONKEY_GIT_URL = "https://github.com/guardicore/monkey"
 # Link to the latest python download or install it manually
-$PYTHON_URL = "https://www.python.org/ftp/python/2.7.13/python-2.7.13.amd64.msi"
+$PYTHON_URL = "https://www.python.org/ftp/python/3.7.4/python-3.7.4-amd64.exe"
```

Comment:
Why 3.7? Is that what you are testing on?
@ShayNehmad this means a version mismatch with the Linux distros

Reply:
Yes. And why not? We didn't talk about any specific python 3 version, so I presumed the latest stable.